src/components/Cart/CartDrawer/CartDrawer.tsx

```
@@ -0,0 +1,39 @@
+'use client';
+
+import { observer } from 'mobx-react-lite';
+import { cartStore } from '../../../stores/cart-store';
+import CartDrawerContainer from './CartDrawerContainer';
+import EmptyCartContent from './CartDrawerEmpty';
+import CartItemList from './CartDrawerList';
+
+const CartDrawer: React.FC = observer(() => {
+  const isOpen = cartStore.drawerOpen;
```

You don't seem to do anything with the open state in this component except pass it down to the next one.

Is it necessary that we pull it here, or can it be pulled at that level?

---

I wanted to keep it separate in case I reuse that drawer, perhaps for mobilemenu when I do that rework.

---

canceled